### ISSUE TITLE
Uninstallation

### ISSUE THREAD

**Issue by Det87** [iccir/Fermata#8]
I uninstalled, because I don't actually want any kind of "polling" be done by the app. I liked InsomniaX in that it does just what the` pmset -b disablesleep` does. You just enable / disable.

Anyways, to uninstall, is it intended to remove the helper manually (`/Library/PrivilegedHelperTools/com.iccir.Fermata.Helper` & `/Library/LaunchDaemons/com.iccir.Fermata.Helper.plist`)?  Are there any other leftovers as well?

**Comment by iccir** [comment]
I think those are the only two files. Out of curiosity, did InsomniaX get updated to not require the disabling of SIP?

**Comment by Det87** [comment]
No, I guess not. The git repo is read-only (https://github.com/semaja2/InsomniaX), and the former homepage is 404 (http://semaja2.net/ye-ol-projects/insomniaxinfo/).

There's also this weird place, where you can submit macOS version test results, and High Sierra seems the last one: https://roaringapps.com/app/insomniax
